notebooks/mobileclip-video-search/mobileclip-video-search.ipynb

```diff
@@ -793,15 +793,12 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #62.        image_search: Image.Image | None,
</pre><p>such syntax supported only for python 3.10+</p><p>image_search: Optional[Image.Image]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Fixed